azure-pipelines.yml

```diff
@@ -71,20 +73,6 @@ stages:
         GOVUK_NOTIFY_CALLBACK_API_KEY: $(govukNotifyCallbackAPIKey)
         SANDBOX: $(sandbox)
 
-    - script: |
-        mkdir $(System.DefaultWorkingDirectory)/docker_images
-        docker save $(docker history -q $(dockerHubUsername)/$(imageName) | grep -v "<missing>") | gzip > $(System.DefaultWorkingDirectory)/docker_images/$(imageName).tar.gz
-        docker images $(dockerHubUsername)/$(imageName) | sed 1d | awk '{print $1 " " $2 " " $3}' > $(System.DefaultWorkingDirectory)/docker_images/images_manifest.txt
-      displayName: 'Export Docker image pipeline artifact'
-      condition: and(eq(variables['buildCancelled'], false), and(succeeded(), eq(variables['deployOnly'], false)))
-
-    - task: PublishPipelineArtifact@1
```

Comment:
How come this is going?

Reply:
this was not working for sometime now when we decided to always publish to dockerhub, all the downstages, pull the docker image from dockerhub instead of from the artifact